.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "external/metal"]
```

Comment:
Do we want to go down the submodule root? If so, should google test be pulled in a similar manner?